### PR TITLE
 feat: added alert component

### PR DESCRIPTION
**Pull request by Bhakti199**
added alerts

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -8,15 +8,16 @@ html,body{
...
     --greyTwo:#57534e;
+    --off-teal:#f0fdfa;
     --teal:#2dd4bf;
     --off-white:#fafaf9;
     --alert-green:#d9f99d;
     --alert-blue: #bfdbfe;
     --alert-yellow:#fde68a;
     --alert-red: #fecaca;
-    --dark-teal: #0d9488;
+    --dark-teal: #115e59;
```
Comment (sks-21) [inline style.css]: You may use generic names for colour so that you can use in different components too.

**Review by sks-21 — APPROVED** [review]
The code looks nice. I would suggest you to prefer relative units over absolute units.

**Comment by Manju-sree** [comment]
Try to use rem unit when there is more than 10px.

File: documentation.html [hunk 2]
```diff
@@ -34,10 +34,10 @@
     <ul class="sidebar" id="sidebarMenu">
         <!-- <li class="sidebar-heading"><a href="" class="ranger-ui  ranger-ui-name">RangerUI</a></li> -->
```
Comment (t007rushi) [inline documentation.html]: Remove unwanted comments in code

File: style.css [hunk 3]
```diff
@@ -67,16 +67,7 @@ html,body{
...
-    position: fixed;
-    top: 20px;
-    left: 100px;
-}
-.fa-bars{
-    font-size: 30px;
-    color: var(--teal);
-    
-}
+
```
Comment (t007rushi) [inline style.css:69]: Remove those spaces & indentation you can use prettier

**Review by t007rushi — COMMENTED** [review]
One more thing host this as there is no preview there

**Pull request merged**